daphne_worker/src/durable/garbage_collector.rs

```diff
@@ -81,9 +81,10 @@ impl DurableObject for GarbageCollector {
                             &(),
                         )
                         .await?;
-                    debug!(
+                    trace!(
                         "GarbageCollector: deleted {} instance {}",
```

Comment:
I'm preeeetty sure this works but not 100%

```suggestion
                        "GarbageCollector: deleted {durable_ref.binding} instance {durable_ref.id_hex}",
```

Reply:
I tried a name with a dot in it earlier today and it was rejected because of the ".".  The doco says it must be an identifier, and I think that something like durable_ref.binding is actually an expression.